### PR TITLE
Refactoring event and file modal

### PR DESCRIPTION
**Pull request by oualid-aloui-rte**
Firstly, I create a "modal" subfolder in the "component" folder, where I will define my EventModal, my FileModal and a ModalHeader which is common to both

Secondly, I replace the code with this one in global-view-core.js and in running-tasks-view-core.js beacause it's the same.

In this way we split the code into components and removed code duplication by reusing the components that we defined only once. In addition, it is a good practice to define the type of properties of our components, which is why I added “Proptypes”.

We thus obtain a code that is more readable, more concise, more reusable and without duplication.

### CODE REVIEW

**Review by schwitzguebel — APPROVED** [review]

File: src/components/modal/event-modal.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```
Comment (kahyami) [inline src/components/modal/event-modal.js:1]: add copyright please

File: src/components/modal/file-modal.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```
Comment (kahyami) [inline src/components/modal/file-modal.js:1]: Add copyright please

File: src/components/modal/modal-header.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```
Comment (kahyami) [inline src/components/modal/modal-header.js:1]: the same here

**Review by kahyami — CHANGES_REQUESTED** [review]

File: src/components/modal/event-modal.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+export default EventModal;
+
+function EventModalContent({ isLoadingEvent, taskData }) {
+    return (
+        <Box sx={style.modalContentStyle}>
+            {isLoadingEvent ? (
+                <LinearProgress />
+            ) : (
+                <EventsTable id="modal-modal-description" taskData={taskData} />
```
Comment (kahyami) [inline src/components/modal/event-modal.js:57]: The modal word is duplicated in the id.

Comment (oualid-aloui-rte) [reply src/components/modal/event-modal.js]: This id was already present before and I think it comes from a copy and paste of the MUI doc and it indeed has no use so I will delete it

**Review by kahyami — APPROVED** [review]

**Pull request merged**